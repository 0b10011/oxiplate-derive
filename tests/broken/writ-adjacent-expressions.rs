use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
#[oxiplate_inline = "{{ foo bar }}"]
struct Data {
    foo: &'static str,
    bar: &'static str,
}

fn main() {
    print!(
        "{}",
        Data {
            foo: "foo",
            bar: "bar"
        }
    );
}
