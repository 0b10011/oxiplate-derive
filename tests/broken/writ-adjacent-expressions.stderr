error: Expecting the writ tag to be closed with `_}}`, `-}}`, or `}}`.
 --> tests/broken/writ-adjacent-expressions.rs:4:28
  |
4 | #[oxiplate_inline = "{{ foo bar }}"]
  |                            ^
